src/Microsoft.AspNetCore.Http.Features/IHttpRequestFeature.cs

```diff
@@ -49,6 +49,11 @@ public interface IHttpRequestFeature
         string QueryString { get; set; }
 
         /// <summary>
+        /// The request target as it was sent in the HTTP request.
+        /// </summary>
+        string RawTarget { get; set; }
```

Comment:
It's the technically correct term :stuck_out_tongue: 

What about RawRequest? Or RawPath?